src/MultiTypeSymbolTable.chpl

```diff
@@ -55,8 +55,16 @@ module MultiTypeSymbolTable
 
             // check to see if userDefinedName is defined
             if (registry.contains(userDefinedName)) {
-                mtLogger.debug(getModuleName(),getRoutineName(),getLineNumber(),
-                                     "regName: redefined symbol: %s ".format(userDefinedName));
+                mtLogger.error(getModuleName(),getRoutineName(),getLineNumber(),
+                                     "regName: requested symbol `%s` is already in use".format(userDefinedName));
+                // We should return an error here
+                throw getErrorWithContext(
+                                    msg=incompatibleArgumentsError("regName", name),
+                                    lineNumber=getLineNumber(),
+                                    routineName=getRoutineName(),
+                                    moduleName=getModuleName(),
+                                    errorClass="ArgumentError");
+                // throw new owned ArgumentError();
```

Comment:
remove this comment?

Reply:
oh, i get the comment now it brackets the throw statement.

Reply:
Doh, you were right, I goofed it, the `throw new owned ArgumentError();` was supposed to come out (in fact I should probably remove the preceding comment as well or else make it more descriptive).

Actually this brings up a question:   I originally had it as throw new owned ArgumentError(), but saw that other places were using the getErrorWithContext proc... However, there was no entry in the `ServerErrorStrings.chpl` for `argumentError`, only `incompatibleArgumentsError`.    Is it ok to use that here or should I just throw the straight ArgumentError ?

Reply:
I think it's ok. unless @hokiegeek2 has something to say about this?